modules/ROOT/pages/upgrade-pre-requisites.adoc

```diff
@@ -0,0 +1,19 @@
+= Upgrade Pre-requisites 
+
+For all OpenShift upgrades from versions 2.6.52 and earlier to versions 2.7.0 and later, you must manually increase the `rtfd` container resources to the desired values.
```

Comment:
@luanamulesoft a quick suggestion for choosing desired values can we provide a hyperlink here as well to the scalability benchmarks page?